### ISSUE TITLE
Unreal Tournament '99 v469c

### ISSUE THREAD

**Issue by PhasecoreX** [luxtorpeda-dev/packages#772]
### Project Link

https://github.com/OldUnreal/UnrealTournamentPatches/releases

### Project License

https://github.com/OldUnreal/UnrealTournamentPatches/blob/master/LICENSE.md

### Steam App Ids

https://steamdb.info/app/13240/

### Additional Context

Current version in this repo is 469b, from over a year ago (https://github.com/luxtorpeda-dev/packages/tree/master/engines/ut99/assets). Recently version 469c was released, supporting 64bit on Linux, among other things.

Wasn't sure how to ask for an update to an existing package, hopefully this is sufficient.

Thank you

**Comment by d10sfan** [comment]
@PhasecoreX Thanks for the issue! I was in the middle of testing so it's now live - https://github.com/luxtorpeda-dev/packages/commit/c705c607c10d56d26b497f7f6e35fe89208aba2e

Let me know if you run into any issues.

**Comment by PhasecoreX** [comment]
Seems to be working! However there was a roadblock since I am using Flatpak. I kept getting this error:

```
./System64/ut-bin: error while loading shared libraries: /home/phasecorex/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/common/Unreal Tournament/linuxdata-469c/System64/Core.so: cannot allocate memory in static TLS block
```

I found [this previous issue](https://github.com/luxtorpeda-dev/packages/issues/725), and came to the solution of changing the Steam launch options to this:

```
LD_AUDIT="" %command%
```

With that, it works fine! Not sure what `LD_AUDIT=""` does exactly, or if it should (could) be added to `run-ut99-469c.sh`. But these are my findings for whoever wants to run it via Flatpak Steam.
